web/packages/teleterm/src/ui/components/FieldInputs.tsx

```diff
@@ -20,7 +20,15 @@ import styled from 'styled-components';
 
 import FieldInput from 'shared/components/FieldInput';
 
-export const ConfigFieldInput = styled(FieldInput).attrs({ size: 'small' })``;
+export const ConfigFieldInput = styled(FieldInput).attrs({ size: 'small' })`
+  input {
+    &:invalid,
+    &:invalid:hover {
+      border-color: ${props =>
+        props.theme.colors.interactive.solid.danger.default};
```

Comment:
This is so that each individual port field can be highlighted in red if the relevant request fails.